### DISCUSSION TITLE
Bounding boxes jumping a lot, causing a lot of false detections

### DISCUSSION THREAD

**Discussion by Jonnehs** [blakeblackshear/frigate#14248] [Detector Support] [answered]
Hi,

Strange issue that I am not sure how to deal with. The bounding boxes around some objects, cars on my street in my case, jump around a lot and get redrawn sometimes two or three times the size of the object in some frames, this often causes them to break (erroneously) into my zones and cause alerts, but particularly problematic is the fact that they keep parked cars in the cameras view from being detected as stationary and its causing a lot of data consumption with recordings that show nothing of interest.

 Just an anecdotal snip:

![image](https://github.com/user-attachments/assets/7a603bef-23e1-47f3-a68f-bf6cafe7742e)
![image](https://github.com/user-attachments/assets/decc29c7-32a6-40ef-a25b-69a80d63692c)

same car, same place, same conditions but the bounding box has obviously enlarged. Anything I can do to improve or prevent this? Can provide my config if needed but its using a coral.ai USB detector for object detection.

Thanks!

**Comment by Jonnehs** [comment]
Forgot to say, this is happening on multiple cameras, even during the day - therefore i assume its a common cause ie config or detector settings etc. Could be something to do with my Reolink detection stream settings, they are pretty high quality though for the fluent channel - 640:480 384kb, 10 fps

**Comment by NickM-27** [comment]
you need to provide your config

**Reply by Jonnehs** [reply]
> you need to provide your config

```mqtt:
  host: 10.25.1.15
  user: mqttuser
  password: mqttpassword
birdseye:
  enabled: true
  width: 2560
  height: 1440
  quality: 8
  mode: continuous
ffmpeg:
  hwaccel_args: preset-vaapi
  input_args: preset-rtsp-restream
  output_args:
    record: -f segment -segment_time 60 -segment_format mp4 -reset_timestamps 1 -strftime
      1 -c:v copy -c:a copy
cameras:
  Front:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.102:554/h264Preview_01_sub
          input_args: preset-rtsp-restream
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.102:554/h264Preview_01_main
          input_args: preset-rtsp-restream
          roles:
            - record
    zones:
      frontgarden:
        coordinates: 
          0.002,0.42,0.122,0.418,0.269,0.411,0.494,0.375,0.661,0.359,0.882,0.292,1,0.425,1,0.999,0,0.999
        inertia: 3
        loitering_time: 0
    detect:
      width: 1280
      height: 720
      fps: 10
    review:
      alerts:
        required_zones: frontgarden
  Garage:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.103:554/h264Preview_01_sub
          input_args: preset-rtsp-restream
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.103:554/h264Preview_01_main
          input_args: preset-rtsp-restream
          roles:
            - record
    detect:
      width: 1280
      height: 720
      fps: 10
  Rear:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.104:554/h264Preview_01_sub
          input_args: preset-rtsp-restream
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.104:554/h264Preview_01_main
          input_args: preset-rtsp-restream
          roles:
            - record
    detect:
      width: 1280
      height: 720
      fps: 10
  Driveway:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.101:554/h264Preview_01_sub
          input_args: preset-rtsp-restream
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.101:554/h264Preview_01_main
          input_args: preset-rtsp-restream
          roles:
            - record
    zones:
      drivewayparked:
        coordinates: 0.222,0.287,0.586,0.286,1,0.468,1,1,0.667,1,0.449,1
        inertia: 3
        loitering_time: 0
        objects:
          - bird
          - cat
          - dog
          - motorcycle
          - person
      driveway:
        coordinates: 0.45,0.285,0.451,0.225,0.208,0.207,0.229,0.287
        inertia: 3
        loitering_time: 0
    detect:
      width: 1280
      height: 720
      fps: 10
    objects:
      filters:
        car:
          mask: 0.073,0.035,0.18,0.038,0.18,0.102,0.071,0.112
    review:
      alerts:
        required_zones:
          - driveway
          - drivewayparked
motion:
  threshold: 40
  contour_area: 40
snapshots:
  enabled: true
  bounding_box: true
  retain:
    default: 21
record:
  enabled: true
  preview:
    quality: very_high
  retain:
    days: 21
    mode: motion
  events:
    pre_capture: 10
    post_capture: 10
    retain:
      default: 21
      mode: active_objects
      objects:
        person: 21
        car: 21
        motorcycle: 21
        dog: 10
        cat: 10
live:
  height: 720
  quality: 5
objects:
  track:
    - person
    - car
    - motorcycle
    - cat
    - dog
    - bird
detectors:
  coral:
    type: edgetpu
    device: usb
version: 0.14
```

**Reply by NickM-27** [reply, marked as answer]
if your reolink cameras support it, using the recommended reolink config would be better as the sub stream would be higher resolution (ext stream is higher res than sub). In general this is just going to be a detection issue which the model and the image are factors of

**Reply by Jonnehs** [reply]
> if your reolink cameras support it, using the recommended reolink config would be better as the sub stream would be higher resolution (ext stream is higher res than sub). In general this is just going to be a detection issue which the model and the image are factors of

the http stream? I couldn't get it working when I tried before, but I can give it another go

**Reply by Jonnehs** [reply]
huh, honestly didnt expect it to improve but in the last 5 mins its been solid, failed back the config to the old method and it was jittering again, failed forward.. solid..

![image](https://github.com/user-attachments/assets/cd30bbb2-f008-4e24-ad4f-655950676d30)

Guess that did the trick, will update all the cameras to the go2rtc streams and see how I get on. 

Thanks bud
